conf/locale/babel_third_party.cfg

```diff
@@ -4,3 +4,6 @@ input_encoding = utf-8
 
 [django: **/template/**.html]
 input_encoding = utf-8
+
```

Comment:
That is definitely confusing! I believe that they are just close enough that Babel works, but I'm not 100% sure.